interface/patient_file/summary/demographics.php

```diff
@@ -749,78 +751,110 @@ function setMyPatient() {
 if ($GLOBALS['patient_id_category_name']) {
     $idcard_doc_id = get_document_by_catg($pid, $GLOBALS['patient_id_category_name']);
 }
+$res=sqlStatement("SELECT option_id,activity FROM list_options WHERE list_id='patient_file_menu' ");
+
+while ($row = sqlFetchArray($res)) {
+    $patient_file_menu[$row['option_id']]=$row['activity'];
+}
```

Comment:
Note you will no longer need option_id stuff since can just use a custom menu or modify the standard menu instead.

Reply:
Yup that is right